### PR TITLE
Estonian language

### PR DESCRIPTION
**Pull request by juhanjuku**
My first try to commit

### CODE REVIEW

**Comment by lightheaded** [comment]
Didn't manage to find the time to fix my pipeline, so monitoring this PR 🤞

File: sentences/et/_common.yaml
Comment (lightheaded) [inline sentences/et/_common.yaml:8]: I think "entity" has been translated to "olem" in other contexts. Weird word, but in my opinion "üksus" isn't much better, I'd recommend keeping with the previous translations for consistency's sake.

File: sentences/et/_common.yaml
Comment (lightheaded) [inline sentences/et/_common.yaml:9]: I'd recommend "Soovi käsitlemisel ilmnes/tekkis viga"

File: tests/et/_fixtures.yaml
Comment (lightheaded) [inline tests/et/_fixtures.yaml:12]: "Garaaž"

**Review by lightheaded — CHANGES_REQUESTED** [review]
Left a few suggestions

File: sentences/et/_common.yaml
Comment (juhanjuku) [reply sentences/et/_common.yaml:8]: First time I see word "olem", which contexts you referring? I checked my doubts with Google translator

Comment (lightheaded) [reply sentences/et/_common.yaml:8]: I'm just thinking if a person has their UI set to Estonian, and is used to seeing this word in settings and throughout the menu, they would understand the error better than if a different word is used.

Screenshot_20230515_082502_Home Assistant

Üksus does seem better than either, indeed. But I would suggest keeping with the HA translation for now and perhaps propose the change in a PR separately so it's changed everywhere at once.

Comment (juhanjuku) [reply sentences/et/_common.yaml:8]: I must admit, I never switched HA to Estonian language. All those new IT words in Estonian language are hard to digest, so I prefer English in UI. 
Still, I really don't like to speak with my home in foreign language. 
I hope, HA voice assistant finally leads to working vocal communication in our native language and I try to give my part. 
First step would be to see Estonian in Supported Languages list. Hopefully more fellow countryman will join then for fine tuning.

Comment (lightheaded) [reply sentences/et/_common.yaml:8]: It's exactly the same for me. I only switched.to English for checking for consistency.

Anyhow, if you prefer not to change, I support the idea of just getting the language added and building on top of that. 

I did too much in my PR and it got stale 😅

Comment (juhanjuku) [reply sentences/et/_common.yaml:8]: I looked your repository and you have done good job. I you don't mind, I try to take parts from there and continue testing.

Comment (lightheaded) [reply sentences/et/_common.yaml:8]: Of course, feel free. I'm currently adjusting to life with 2 kids,. youngest being 2 weeks old, so I've put my PR on hold 😅

**Comment by juhanjuku** [comment]
I understood, language changes are merged in main releases, but I can't see 'et' added?
I there still anything expected from me?

**Comment by lightheaded** [comment]
It has to be approved from a maintainer. My duplicate PR #841 was somehow assigned to @synesthesiam. Not sure how the PRs get this attention, though. @synesthesiam, perhaps you can take this one as well?

**Comment by juhanjuku** [comment]
Reviewed all my changes in Code Space. All tests passed. New pull request.

**Comment by synesthesiam** [comment]
Hmmm...seems to be stuck in the CI for some reason.

**Comment by juhanjuku** [comment]
> Hmmm...seems to be stuck in the CI for some reason.

What is CI?
Also, I can't understand branch conflict in languages.yaml
What I can do?

**Comment by tetele** [comment]
> What is CI?
Continuous Integration, i.e. automatic testing.

> Also, I can't understand branch conflict in languages.yaml
It means that there have been commits in the `main` branch of the `intents` repo that are conflicting with your code. There are multiple ways of solving these conflicts, there's a link just above the comments box on Github ("_This branch has conflicts that must be resolved. **Use the command line** to resolve conflicts before continuing._")

**Comment by juhanjuku** [comment]
I can't understand where is conflict? In my version are added 2 lines
et:
   nativeName: Eesti
All rest is same as main branch?

**Comment by tetele** [comment]
> As there is also CI problem, is it better if I close this Pull request and make new one?

Push a merge resolve commit and we'll see if the CI pipeline works/starts on the new commit

**Comment by juhanjuku** [comment]
> That's how Git works. https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-using-the-command-line

Followed your advise,
cloned my repository to local machine and run $ git status,
but can't see any conflicts?

**Comment by tetele** [comment]
> but can't see any conflicts?

That's because you haven't pushed the conflicting code so that you can clone it locally. Cloning on the local machine and adding modifications should produce merge conflicts, not solve ones that are created elsewhere.

**Comment by juhanjuku** [comment]
> > but can't see any conflicts?
> 
> That's because you haven't pushed the conflicting code so that you can clone it locally. Cloning on the local machine and adding modifications should produce merge conflicts, not solve ones that are created elsewhere.

Too complicated to me. I'm closing this pull request, delete my intents repository from github and start from zero with CodeSpace

**Pull request closed without merging**
